content/pt/docs/concepts/resources/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Resources
+weight: 70
+default_lang_commit: a1740fd934e595f1396f2eb82a58a80824369b09
+---
+
+## Introdução
+
+{{% docs/languages/resources-intro %}}
+
+Se você usar [Jaeger](https://www.jaegertracing.io/) como seu backend de
```

Comment:
```suggestion
Se você usar [Jaeger](https://www.jaegertracing.io/) como seu _backend_ de
```

Reply:
I don't see the value in using _emphasis_ in docs like this. Better not to.

Reply:
I guess "backend" is current accepted jargon among the dev community though, isn't it?

Reply:
Yup! We tried to find a translation, but it ended up more confusing and cryptic since "backend" is more commonly used.

Reply:
In that case, I'd treat it as a non-foreign word. My two cents!

Reply:
Yeah, that's a good point. Some style guides do have list of foreign words that are not italicized (for example https://www12.senado.leg.br/manualdecomunicacao/verbetes-acessorio/estrangeirismos-grafados-sem-italico-ou-aspas). 

I think the challenge for us would be to apply them consistently without automation. Having "all foreign words should be italicized" is easier to remember and apply than "all foreign words should be italicized, except for..." 😅 

Maybe this is something `textlint` could help us.